classes/XLite/Module/ShipStation/Api/changelog/5.4/1/2.log

```diff
@@ -1,2 +0,0 @@
-Version 5.4.1.2
-02/10/2021 - [Changes] Import "company name" field with Orders
```

Comment:
Instead of deleting these, why not move them to `modules/ShipStation/Api/docs/version_info`? 
https://developer.x-cart.com/migration_guides/module_changelogs#where-to-put-changelogs